pages/plugins/using.md.erb

```diff
@@ -0,0 +1,131 @@
+# Using Plugins
+
+Plugins can be used in any pipeline [command steps](/docs/pipelines/command-step) that you’ve defined using YAML.
+
+<%= toc %>
+
+## Adding a plugin to your pipeline
+
+To add a plugin to a [command step](/docs/pipelines/command-step), use the `plugins` attribute.  The `plugins` attribute accepts an array, so you can add as many plugins as you require to a step. The plugins will be run in the order you list them in the array. 
+
+In the example below, the `shellcheck` plugin will run, followed by the `docker` plugin:
+
+```yml
+steps:
+  - command: yarn install && yarn run test
+    plugins:
+      - shellcheck#v1.1.2:
```

Comment:
I was hoping you wouldn't bring this up! Our recommendation is actually the less secure version. We should raise this for discussion, because I'd like our default to also be more secure (for reference, see the discussion in here: https://github.com/buildkite/agent/pull/1025).

I think for particularly security conscious customers, customer's security teams, or people that build their own pipeline tooling… they'll want to use digests. Hence why it's only in the securing plugins guide, and nowhere else.

We want to get to a place where the default is secure though.

Reply:
I just wrote a clarifying comment on that PR. I think we should probably reword these new docs to not push the commit SHA version so hard, keep the current examples as-is, and plan to fix it at a platform level.

Reply:
I've just pushed some new words in 4b72cbe — would love your edits, because they're not the best words.